Modules/Loadable/Markups/MRML/vtkMRMLMarkupsNode.h

```diff
@@ -264,6 +269,7 @@ class  VTK_SLICER_MARKUPS_MODULE_MRML_EXPORT vtkMRMLMarkupsNode : public vtkMRML
 
   /// Clear out the node of all control points
   virtual void RemoveAllControlPoints();
+	virtual void UnsetAllControlPoints();
```

Comment:
It looks like your indentation is messed up in several places where it has become overindented. This comment is just pointing out one location, but appears to be happening across several files in this PR.

Reply:
Thanks @jamesobutler  I seem to have a had an issue with my editor, I'll take a look.